VideoLocker/src/main/java/org/edx/mobile/task/GetCommentsListTask.java

```diff
@@ -25,8 +25,15 @@ public GetCommentsListTask(@NonNull Context context, @NonNull String responseId,
     }
 
     public Page<DiscussionComment> call() throws Exception {
-        List<String> requestedFields = Collections.singletonList(
-                DiscussionRequestFields.PROFILE_IMAGE.getQueryParamValue());
+        final List<String> requestedFields;
+        {
```

Comment:
It might come in handy in the future, thats what I actually had in mind. Like different case for different required fields addition in the list.